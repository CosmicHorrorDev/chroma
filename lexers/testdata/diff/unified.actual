--- old.txt	2023-01-17 20:59:02.687400701 -0700
+++ new.txt	2023-01-17 21:00:40.760132465 -0700
@@ -1,4 +1,7 @@
-A
+
+
+
 B
-C
+F
 D
+E
