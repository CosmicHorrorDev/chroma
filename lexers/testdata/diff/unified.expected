[
  {"type":"GenericDeleted","value":"--- old.txt\t2023-01-17 20:59:02.687400701 -0700\n"},
  {"type":"GenericInserted","value":"+++ new.txt\t2023-01-17 21:00:40.760132465 -0700\n"},
  {"type":"GenericSubheading","value":"@@ -1,4 +1,7 @@\n"},
  {"type":"GenericDeleted","value":"-A\n"},
  {"type":"GenericInserted","value":"+\n+\n+\n"},
  {"type":"Text","value":" B\n"},
  {"type":"GenericDeleted","value":"-C\n"},
  {"type":"GenericInserted","value":"+F\n"},
  {"type":"Text","value":" D\n"},
  {"type":"GenericInserted","value":"+E\n"}
]
